kafkaesk/app.py

```diff
@@ -496,7 +496,7 @@ async def on_message(record: aiokafka.structs.ConsumerRecord) -> None:
         await self.stop()
 
         # re-raise any errors so we can validate during tests
-        for task in done | pending:
```

Comment:
what's the implication of this change? is it that we don't care about `pending` so no need to check for exceptions there?

Reply:
You can't call `exception` on a pending task -- it will raise exception.

Reply:
yeah, there shouldnt be any pending after the `self.stop()`, but if there is any there is no point into try to getting the exception from an unfinished task since thta will raise an exception